tests/test_cli.py

```diff
@@ -83,21 +69,12 @@ def test_parse_argv_no_options(tmpdir, option):
 
 
 @pytest.mark.parametrize('verbose', (True, False))
-def test_log_verbosity(runner, verbose):
+def test_log_verbosity(verbose, recipe_arguments):
     """Ensure that the verbosity is set properly on."""
-
-    # Initial state – if the test fails here, the app has changed
-    # and the test needs to be adjusted.
-    assert LOG.getEffectiveLevel() == logging.INFO
-
-    recipe = Path('recipe.yml')
-    recipe.touch()
-
-    verbose_args = ['--verbose'] if verbose else []
-    level = logging.DEBUG if verbose else logging.INFO
-
-    runner.invoke(run, verbose_args + [str(recipe), 'test'])
-    assert LOG.getEffectiveLevel() == level
```

Comment:
Because I thought the `test_log_verbosity` test was for below area. And I moved the callback logic from here to `run` function.
But if you like keeping the logic here. I will keep. I will rebase.

```
@click.option(
    '--verbose', '-v', is_flag=True, default=False,
    help='Turn on verbose logging.',
)
```